### ISSUE TITLE
Why still old Syntax?

### ISSUE THREAD

**Issue by somsharan** [Polymer/designer#206]
I am not sure whether i need to ask this here or not.

But Still, Why does polymer designer/Chrome Dev Editor still does not use syntax of polymer 1.0 and above.

New To Polymer.js

**Comment by addyosmani** [comment]
https://github.com/Polymer/designer/blob/master/elements/designer-frame/designer-frame.html and so on are using the 1.0 syntax. Where are you seeing older syntax being used in the designer specifically

**Comment by somsharan** [comment]
https://polymer-designer.appspot.com/ tool still has old syntax..

**Comment by justinfagnani** [comment]
This is true, the live site is very old now, but I've been resisting taking it down so that people still have access to their old projects. Maybe it's not that important though, as they still have them in gists.
